config/settings.yml

```diff
@@ -76,6 +76,10 @@
   :concurrent_per_host: 1
   :scan_via_host: true
   :use_vim_broker: true
+:container_logging:
+  :by_provider_name:
+    :_default:
+      :kibana_route: logging-kibana
```

Comment:
@simon3z what do you think of that as a solution for settings per-provider? (I can also use that in other places)
cc @alongoldboim thanks for the idea!